### PR TITLE
Updates Null Rod menu to TGui

### PR DESCRIPTION
**Pull request by Moltijoe**
with how many null rods we have, selecting is annoying and tedious

also moves all armours into their own file

https://user-images.githubusercontent.com/108117184/222194575-0979d1ac-70d5-40f4-9771-6d5149c69579.mp4

:cl: @Chubbygummibear
tweak: Null Rod selection uses TGui
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
masochism

**Comment by Moltijoe** [comment]
> masochism

honestly, probably wouldn't be so bad if i knew what i was doing, but i have only about half a class worth of javascript experience

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
The capitalization on these is kinda whack, the list having different capitalization vs the actual item + the tabs

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -273,80 +50,86 @@
...
-
-/obj/item/nullrod/godhand/ignition_effect(atom/A, mob/user)
-	. = span_notice("[user] grasps [A] with [user.p_their()] flaming hand, igniting it in a burst of holy flame. Holy hot damn, that is badass. ")
-
+	if(.)
+		return
+	var/mob/user = usr
+	switch(action)
+		if("confirm")
+			var/rodPath = text2path(params["rodPath"])
```
Comment (ThatLing) [inline code/game/objects/items/holy_weapons.dm:118]:
```suggestion
			var/rodPath = text2path(params["rodPath"])
			if(!ispath(rodPath))
				return FALSE
```

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Review by ToasterBiome — APPROVED** [review]

**Review by ThatLing — APPROVED** [review]

**Pull request merged**